README.md

```diff
@@ -33,6 +33,19 @@ Application Options:
   --sign-args                 Extra arguments to use when signing with gpg[2]
 ```
 
+
+### Environment variables
+
+Some of the parameters above can alternatively be specified as environment variables. Also, some additional environment variables are available, too.
+
+- `ARCH`: Needs to be set whenever appimagetool cannot automatically determine the architecture of the binaries inside the AppDir to choose a suitable runtime (e.g., when binaries for multiple architectures or just shell scripts are contained in there).
+- `APPIMAGETOOL_FORCE_SIGN`: By default, if signing fails, appimagetool just logs a warning but will not abort immediately. If this environment variable is set, appimagetool exits with a non-zero return code.
+- `APPIMAGETOOL_SIGN_PASSPHRASE`: If the `--sign-key` is encrypted and requires a passphrase to be used for signing (and, for some reason, GnuPG cannot be used interactively, e.g., in a CI environment), this environment variable can be used to safely pass the key.
+- `GITHUB_TOKEN`: If set to any value, appimagetool will try to guess some update information to the current repository's release page. Note that this guessing is opinionated and requires the releases to be set up in a specific way. If it is *not* set up, AppImageUpdate will try to update but always fail.
```

Comment:
This is an incomplete description. And the process truly is opinionated. For instance, the conventions about `continuous` and `latest`.

By the way, you could use the suggestions feature.

Reply:
The suggestions feature... where is that...

Certainly we should document how the "educated guess" (using conventions) works.
(This is complex and one can mess up easily. For this reason, go-appimage does the uploading for the user in one step, so that the user has no chance to mess it up.)

Reply:
![screenshot_2023-07-05_23-56-04](https://github.com/AppImage/appimagetool/assets/4503202/2ad31f0b-8b3a-4192-9169-1623afe11304)